### DISCUSSION TITLE
Is it time for Reliability features to be the default for Sidekiq Pro?

### DISCUSSION THREAD

**Discussion by swrobel** [sidekiq/sidekiq#5931] [Ideas]
[super_fetch](https://github.com/sidekiq/sidekiq/wiki/Reliability#using-super_fetch) has been around since Sidekiq Pro 3.4 and it seems reasonable to me that most other Pro users would want and expect this behavior by default. It seems there are [other users](https://github.com/sidekiq/sidekiq/discussions/5848#discussioncomment-5431156) who were caught off guard by the fact that this needs to be enabled manually. It seems at some point there was a `config.reliable!` option that would enable all of these, but the [reliability features](https://github.com/sidekiq/sidekiq/wiki/Reliability) seem to require a bit more config to get them all working now.

Is there any reason not to make reliable features the default?

**Comment by mperham** [comment]
I appreciate your prompt to rethink the defaults.

The commercial packages have always started dormant: you have to activate any features you wish to use. This is because I don't want the user to see any behavioral or semantic changes in Sidekiq just from adding a gem. I really don't think it's wise to change this ethos at this point. Customers are expected to read the documentation for the products they purchase so they know how to use them properly. If they don't, that's their choice.

Would it be reasonable to start printing a warning if they are using Sidekiq Pro but don't have reliability enabled? Maybe we could add a new `config.unreliable!` helper which tells us the user does not want super_fetch and disables the warning.

More ideas welcome!
